.firebase/hosting.ZGlzdA.cache

```diff
@@ -0,0 +1,16 @@
+index.html,1576017182418,0afb5673e9b5941dd222b9c2ce2eef1b601ee74e8112e37a9180261af3a6414c
```

Comment:
Please delete this file so it does not get merged in, and ensure .firebase is in your gitignore for this branch so this does not get accidentally committed.